src/core/SkipToContent/SkipToContentLink.tsx

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { CommonProps, useTheme } from '../utils';
+import '@itwin/itwinui-css/css/skip-to-content.css';
+
+export type SkipToContentLinkProps = {
+  /**
+   * Localize 'Skip to main content' label if needed.
+   * @default 'Skip to main content'
+   */
+  children?: React.ReactNode;
+} & Omit<CommonProps, 'href'>;
```

Comment:
This should support all anchor props, including `href`, to provide the most flexibility to user.

```suggestion
& React.ComponentPropsWithoutRef<'a'>;
```

Reply:
Done